app/components/moj_forms/settings_screen_component.rb

```diff
@@ -1,6 +1,6 @@
 module MojForms
   class SettingsScreenComponent < GovukComponent::Base
-    renders_one :back_link, MojForms::BackLinkComponent
+    # renders_one :back_link, MojForms::BackLinkComponent
```

Comment:
do we still want this?

Reply:
No, we don't need it anymore.  I'll remove the comment